docs/policies.rst

```diff
@@ -393,5 +394,78 @@ It is recommended to use
           ``mu_neg = mu_pos`` and ``use_max_sim_neg = False``. See
           `starspace paper <https://arxiv.org/abs/1709.03856>`_ for details.
 
+Two Stage Fallback Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This policy handles low NLU confidence in multiple stages.
+
+- If a NLU prediction has a low confidence score, the user is asked to confirm
+  whether they really had this intent.
```

Comment:
@MetcalfeTom "had this intent" doesn't sound right to me, would you say "the user is asked to confirm the classification of the intent" is better?